crates/oxc_linter/src/snapshots/prefer_tag_over_role.snap

```diff
@@ -0,0 +1,35 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: prefer_tag_over_role
+---
+  ⚠ eslint-plugin-jsx-a11y/prefer-tag-over-role: Prefer using semantic HTML tags over `role` attribute.
```

Comment:
The snapshot looks incorrect

Reply:
Updated snaspshot! How about this one...? 👀 
[update snapshot](https://github.com/biomejs/biome/pull/1248/commits/4680e6ad5dc48d1a8dd9345bc98d83ab0a6dbba6)